test/custom-fs.js

```diff
@@ -16,7 +16,27 @@ function intercept (method, path) {
   intercepted._removed = intercepted._removed.sort()
 }
 
-var expectAsync = {
+function makePaths(expected) {
+  if (process.platform !== 'win32')
+    return expected
+  function makePath(path) {
+    return path.replace(/\//g, '\\');
+  }
```

Comment:
Thanks @bzoz! 

Could you move `makePath` outof `makePaths`.
Instead of wrapping the entire `expectAsync` could you apply the path normalization per sub object, e.g. 
```js
_removed: makePaths([...]),
_saved: makePaths([...]),
```
That way it's easier to apply to other expected objects as tests are added.